docs/vaadin-upload-overview.adoc

```diff
@@ -0,0 +1,23 @@
+---
+title: Overview
+order: 1
+layout: page
+---
+
+[[vaadin-upload.overview]]
+= Overview
+
+The [elementname]#vaadin-date-picker# is a Polymer element for uploading multiple files to server.
+
+It allows users to add their files by dropping them directly into the component. Also it has the “Add files” button, which opens the native browser file dialog for selecting the file or files to process.
+
+All files added by user are displayed in a file list inside the element.
+
+The added files would be automatically uploaded to server. The uploading of multiple files is performed simultaneously. The list indicates the state and uploading progress for each file.
+
+Each file could be removed from the list by user by pressing the corresponding “×” icon. In this case, if the file uploading has been in progress, then the upload process would be aborted.
```

Comment:
I would not say "x" icon (avoid describing visual styles in the documentation). Instead, try "by pressing the corresponding cancel button".